src/api/cow/index.ts

```diff
@@ -26,6 +26,8 @@ import {
 import { CowError, logPrefix, objectToQueryString } from '../../utils/common'
 import { Context } from '../../utils/context'
 
+export { OperatorError, QuoteError }
+
```

Comment:
This doesn't seem related to the change in question.
What is this for/from?

Reply:
did some tests and forgot to remove it, thanks for catching it up.